### ISSUE TITLE
Missing keys in state_dict

### ISSUE THREAD

**Issue by georgepar** [tenexcoder/huggingface-tutorials#1]
Hi,
I'm trying to replicate your result with the performer and I'm facing the following issue:

First I run `python hf_port.py` to create `distilgpt2.pt` in a format compatible with performer and then I run `python load.py` to test the model loading.

I end up with the following trace

```
        Missing key(s) in state_dict: "performer.net.blocks.0.f.net.fn.fast_attention.calls_since_last_redraw", "performer.net.blocks.0.f.net.fn.fast_attention.projection_matrix", "performer.net.blocks.1.f.net.fn.fast_attention.calls_since_last_redraw", "performer.net.blocks.1.f.net.fn.fast_attention.projection_matrix", "performer.net.blocks.2.f.net.fn.fast_attention.calls_since_last_redraw", "performer.net.blocks.2.f.net.fn.fast_attention.projection_matrix", "performer.net.blocks.3.f.net.fn.fast_attention.calls_since_last_redraw", "performer.net.blocks.3.f.net.fn.fast_attention.projection_matrix", "performer.net.blocks.4.f.net.fn.fast_attention.calls_since_last_redraw", "performer.net.blocks.4.f.net.fn.fast_attention.projection_matrix", "performer.net.blocks.5.f.net.fn.fast_attention.calls_since_last_redraw", "performer.net.blocks.5.f.net.fn.fast_attention.projection_matrix".   
```

So the fast attention matrices are not initialized. Are these the random projection matrices?

I tried with `strict=False` and things start to work, but the loss is very high during the initial finetuning (equivalent with completely random initialization).

Any ideas?
